Makefile

```
@@ -288,6 +295,15 @@ OPERATOR_SDK = $(shell which operator-sdk)
 endif
 endif
 
+.PHONY: api-docs
+API_DOCS_PATH = ./docs/api-overview.md
+
+api-docs: crd-ref-docs ## Creates API docs using https://github.com/elastic/crd-ref-docs
+	mkdir -p docs
+	$(CRD_REF_DOCS) --source-path ./ --output-path $(API_DOCS_PATH) --renderer markdown --config ./crd-ref-docs.config.yaml
+	@# Remove unwanted .io links and clean up blank lines
+	@sed -i.bak -e '/\.io\/[^v][^1].*)/d' -e '/^$$/N;/^\n$$/D' -e '${/^$$/d}' $(API_DOCS_PATH) && rm $(API_DOCS_PATH).bak
```

```suggestion
	@sed -i.bak -e '/\.io\/[^v][^1].*)/d' -e '/^$$/N;/^\n$$/D' -e '${/^$$/d}' $(API_DOCS_PATH)
	rm -f $(API_DOCS_PATH).bak
```

For readability again :)